tests/test_list_columns.py

```
@@ -0,0 +1,93 @@
+
+# ---------- D：删除 / 在...显示 进入 ⋯ ----------
+
+def test_delete_and_reveal_in_more_menu(live_server, browser):
+    pg = _open(live_server, browser)
+    labels = pg.eval_on_selector_all(
+        '.list-view .file-item',
+        """items => {
+            const row = items.find(el => el.querySelector('.file-name-col').innerText.includes('漫画B'));
+            return [...row.querySelectorAll('.more-dropdown .more-item')].map(a => a.textContent.trim());
+        }""")
+    assert any('删除' in l for l in labels), f'⋯ 里应有删除: {labels}'
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)